### ISSUE TITLE
_created_at and _updated_at issue

### ISSUE THREAD

**Issue by Jahoe** [ParsePlatform/parse-server#2203]
Our Parse interface has two fields that have the type String. They are always undefined (they aren't used once in our set up). In our backend they appear as Date fields. When I try and push an update from our iOS app I get the following error:

```
[Error]: invalid field name: _created_at (Code: 105, Version: 1.13.0)
[Error]: invalid field name: _updated_at (Code: 105, Version: 1.13.0)
```

We've tried to use the Parse interface to delete the column but nothing happens. We've also tried deleting them in the backend, despite not being able to see the column.

As far as we know, the columns called createdAt and updatedAt point to their own hidden fields called _updated_at and _created_at. These shouldn't be visible in the interface though. Is there anyway to delete these pesky two columns that are causing a conflict with the hidden _updated_at and _created_at?

[You can see the columns here.](http://i.stack.imgur.com/sNWXT.png)

**Comment by TylerBrock** [comment]
Hey Jahoe, thanks for reporting the issue, what version of parse-server are you running?

**Comment by Jahoe** [comment]
We're running the server on 2.2.12 and I've got the 1.13.0 Parse SDK. After a lot of tinkering we realised that the Javascript SDK allows the saving of individual cells rather than the whole document. This meant we could save the cell without that specific field in the document being populated. However the iOS SDK doesn't accept it, presumably when using .saveEventually() it's actually saving the whole document with the edited cell? 

We've actually done a slight change of plan and now send a request to our EC2 instance to change the value server side with a simple MongoDB request.

We're still unable to change drop the field of _updated_at and _created_at. Seems very interesting.
